tests/foreman/api/test_template.py

```diff
@@ -296,15 +295,13 @@ def test_positive_delete(self):
                 with self.assertRaises(HTTPError):
                     entities.ConfigTemplate(id=c_temp.id).read()
 
-    @tier2
+    @tier1
```

Comment:
Similar to comment above - yes, this is Functional test, but since it's on Acceptance level, it belongs to tier3. Functional tests on integration level can be tier2, and functional test on component level can be tier1 (if they also have Importance: Critical).

Reply:
I think, I need a change in testlevel here in this case.

Reply:
Updated Metadata as per your Doc.